tests/integration/test_grpc_protocol/test.py

```diff
@@ -366,40 +366,37 @@ def test_progress():
         result.time_zone = ""
         result.query_id = ""
     # print(results)
-    assert (
-        str(results)
-        == """[output_format: "TabSeparated"
-progress {
+    expected_results = """[progress {
   read_rows: 2
   read_bytes: 16
   total_rows_to_read: 8
 }
-, output: "0\\t0\\n1\\t0"
+output_format: "TabSeparated"
+, output: "0\\t0\\n1\\t0\\n"
```

Comment:
It seems we've changed how the result is sent - now the first `progress` is sent before `output_format`, and also an end-of-line character is attached to the end of each result block, not to the beginning of next result block. It doesn't really matter.